updater/spec/dependabot/integration_spec.rb

```diff
@@ -37,6 +37,9 @@
     allow(Dependabot::Environment).to receive(:token).and_return("some_token")
     allow(Dependabot::Environment).to receive(:job_id).and_return(job_id)
     allow(Dependabot.logger).to receive(:info).and_call_original
+    message_builder = double(Dependabot::PullRequestCreator::MessageBuilder)
+    allow(Dependabot::PullRequestCreator::MessageBuilder).to receive(:new).and_return(message_builder)
+    allow(message_builder).to receive(:message).and_return(nil)
```

Comment:
❓ Since this is an integration spec, do we want to return `nil` here? I could see value coming from ensuring that this is exercised.

I'm fine with this being added in a separate PR if you think it's valuable. I'll trust your judgment on whether it should be 🙂

Reply:
We're testing it in the smoke tests now pretty extensively. I'm leaning towards not testing it here too 🤔